packages/web-client/tests/acceptance/issue-prepaid-card-test.ts

```diff
@@ -27,8 +30,14 @@ function milestoneCompletedSel(milestoneIndex: number): string {
 
 module('Acceptance | issue prepaid card', function (hooks) {
   setupApplicationTest(hooks);
+  setupMirage(hooks);
 
   test('Initiating workflow without wallet connections', async function (assert) {
+    this.server.db.loadData({
+      prepaidCardColorSchemes: prepaidCardColorSchemes,
+      prepaidCardPatterns,
+    });
```

Comment:
Doesn't matter since we have only one test in this module currently, but this should probably be in a `beforeEach`